docker-compose-image-tag.yml

```diff
@@ -0,0 +1,101 @@
+#
```

Comment:
this file is effectively the old `docker-compose-non-dev.yml` just renamed to be more clear